CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Add `BACKUP_MONGODB_AUTHENTICATION_DATABASE` to override default
 authentication database name.
 * [refactor] Remove obsolete scripts.
 * [fix] Use ephemeral volumes in K8s for data volume
```

Comment:
This has nothing to do with this particular change, but when you follow up with the release patch please be sure to fix the changelog here: e1d733928cdf4c38e42217f4a68ebf80f15010a5 was a feature commit, not a fix. :)

Reply:
OK